L1Trigger/L1TMuonBarrel/src/Twinmux_v1/L1ITMuonBarrelPrimitiveProducer.cc

```diff
@@ -60,8 +60,8 @@ inline L1ITMuonBarrelPrimitiveProducer::~L1ITMuonBarrelPrimitiveProducer()
 {
 }
 
-inline L1ITMuonBarrelPrimitiveProducer::L1ITMuonBarrelPrimitiveProducer( std::unique_ptr<MBLTContainer> _mbltContainer )
-: mbltContainer(std::move(_mbltContainer))
+inline L1ITMuonBarrelPrimitiveProducer::L1ITMuonBarrelPrimitiveProducer( std::shared_ptr<MBLTContainer> _mbltContainer )
```

Comment:
It looks like you copied this code from 80x.
In 81x all the code was migrated to use unique_ptr, instead of the deprecated shared_ptr.